pkg/runners/cairo_runner.go

```diff
@@ -3,6 +3,7 @@ package runners
 import (
 	"errors"
 
+	builtinrunner "github.com/lambdaclass/cairo-vm.go/pkg/runners/builtin_runner"
```

Comment:
For clarity, we can move the builtin to a new `builtins` folder

Reply:
Done!